README.md

```diff
@@ -20,9 +20,9 @@ You will need to install it thanks to the `pip` binary.
 
 # Run the tests
 
-## Using a network configuration file
+The network configuration must be defined in a yaml file.
 
-This is the mode used by default. Two files are currently available:
+Two configurations are currently available:
```

Comment:
What about "configuration files" instead of "configurations"?

Reply:
Thanks.